api.js

```diff
@@ -50,7 +50,7 @@ function getAnswers(question_id, params) {
   return handleGetRequests(`qa/questions/${question_id}/answers`, params)
 }
 
-// Handles all put requests, requires a route and params object
+// Handles all POST requests, requires a route and params object
 function handlePostRequests(route, params) {
```

Comment:
For some of my posts I'll need to include more than just params. I'll need to pass in an object to the assign to the body property of the request.

Reply:
Ya if you need to reformat it in a way that works for you go ahead. I havent actually done any Post request functionality yet but I will soon.